project/src/test/java/com/google/sps/DatabaseTest.java

```diff
@@ -111,8 +111,8 @@ public void testCreateDocument() {
     Entity docEntity = ds.prepare(documentQuery).asSingleEntity();
     Assert.assertEquals(name, (String) docEntity.getProperty("name"));
     Assert.assertEquals(language, (String) docEntity.getProperty("language"));
-    ArrayList<Long> userIDs = (ArrayList) docEntity.getProperty("userIDs");
-    Assert.assertTrue(userIDs.contains(userID) && userIDs.size() == 1);
+    //ArrayList<Long> userIDs = (ArrayList) docEntity.getProperty("userIDs");
```

Comment:
delete what was commented out or change these lines to 
Assert.assertEquals(ownerID, ...)
where ... is  docEntity.getProperty("ownerID")

Reply:
right I still need to go back and finish up the unit tests with new tests